app/models/get_an_identity/access_token.rb

```diff
@@ -0,0 +1,21 @@
+module GetAnIdentity
+  class AccessToken
+    attr_reader :token
+
+    def initialize
+      client_id = ENV.fetch("TRA_OIDC_CLIENT_ID")
```

Comment:
How do we normally handle env variables like this? ie storing them on server more for my information

Reply:
They are stored in Github secrets then the Github Actions pass them into the `cf push` call in each deployment using the --var parameter. This then gets picked up in the manifests and set as an env variable on the server. Which is picked up here. 

These ones are already all set up. :)